.github/workflows/windows-e2e.yml

```diff
@@ -7,222 +7,23 @@ on:
-            -v $PWD/pull-secret:/opt/crc/pull-secret:Z \
-            -v $PWD/output:/output:Z \
-            -v $PWD/id_rsa:/opt/crc/id_rsa:Z \
-            quay.io/crcont/crc-e2e:gh \
-              -targetFolder ${TARGET_FOLDER} \
-              -junitResultsPath ${TARGET_FOLDER}/junit
-
-          # Check logs 
-          podman logs -f crc-e2e
-
-      - name: Test Report
-        id: test-report
-        uses: mikepenz/action-junit-report@v4
-        if: always() # always run even if the previous step fails
-        with:
-          fail_on_failure: true
-          include_passed: true
-          detailed_summary: true
-          require_tests:  true
-          report_paths: '**/*.xml'
-
-      - name: Upload e2e results
-        uses: actions/upload-artifact@v4
-        if: always()
-        with:
-          name: E2E-results-windows-${{ matrix.windows-version }}${{ matrix.windows-featurepack }}
-          path: |
-            **/*.xml
-            **/*.results
-            **/*.log
-
-      - name: Update status of the PR check
-        if: always()
-        run: |
-          set -xuo
-          OUTCOME="success"
-          if [[ ${{steps.test-report.outcome}} != "success" ]]; then 
-            OUTCOME="failure";
-          fi
-
-          DESCRIPTION="Finished"
-          CONTEXT="ci/gh/e2e/windows-${{ matrix.windows-version }}-${{ matrix.windows-featurepack }}"
-
-          # post result to commit status
-          curl -L -v \
-          -X POST \
-          -H "Accept: application/vnd.github+json" \
-          -H "Authorization: Bearer ${{ github.token }}" \
-          https://api.github.com/repos/${{ github.repository }}/statuses/${{ env.COMMIT_SHA }} \
-          -d "{\"state\":\"${OUTCOME}\", \"description\":\"${DESCRIPTION}\", \"context\":\"${CONTEXT}\", \"target_url\":\"https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}\"}"
-
-      - name: Destroy instance
-        if: always()
-        run: |
-          # Make sure lock is removed
-          rm -rf .pulumi/locks/*
-          
-          # Destroy instance
-          podman run -d --name windows-destroy --rm \
-            -v ${PWD}:/workspace:z \
-            -e ARM_TENANT_ID=${{ secrets.ARM_TENANT_ID }} \
-            -e ARM_SUBSCRIPTION_ID=${{ secrets.ARM_SUBSCRIPTION_ID }} \
-            -e ARM_CLIENT_ID=${{ secrets.ARM_CLIENT_ID }} \
-            -e ARM_CLIENT_SECRET='${{ secrets.ARM_CLIENT_SECRET }}' \
-            quay.io/rhqp/qenvs:v0.6.0 azure \
-              windows destroy \
-              --project-name 'windows-desktop' \
-              --backed-url 'file:///workspace'
-          # Check logs
-          podman logs -f windows-destroy
+        qe-type: ['e2e','integration']
+        preset: ['openshift', 'microshift', 'all']
+        exclude:
+        - qe-type: 'e2e'
+          preset: 'all'
+        - qe-type: 'integration'
+          preset: 'openshift'
+        - qe-type: 'integration'
+          preset: 'microshift'
```

Comment:
so does that mean `e2e openshift` and `e2e microshift` but `integration all` ?

Reply:
yeah I did not investigate if integration has labels to allow me to differentiate, it that is the case I can change that back and separate the execution. In any case if possible I will do it on a following I do not want to mess this one more.

As you see the e2e and integration images build are on the windows action which does not make sense to me, I would prefer having a buidler action which builds everything we need (with different jobs, ,as they will run in parallel) and the use that builder action as the trigger for the worklflow run where tests will be executed. But again my idea is having a following for this refactor